mlfinlab/tests/test_labeling_raw_return.py

```diff
@@ -0,0 +1,76 @@
+# pylint: disable=missing-module-docstring
+
+import unittest
+import os
+import numpy as np
+import pandas as pd
+
+from mlfinlab.labeling.raw_return import raw_return
+
+
+class TestLabelingRawReturns(unittest.TestCase):
+    """
+    Tests for the raw returns labeling method.
+    """
+
+    def setUp(self):
+        """
+        Set the file path for the sample dollar bars data.
+        """
+        project_path = os.path.dirname(__file__)
+        self.path = project_path + '/test_data/stock_prices.csv'
+        self.data = pd.read_csv(self.path, index_col='Date')
+        self.idx5 = self.data[:5].index
+        self.col5 = self.data.iloc[:, 0:5].columns
+
+    def test_dataframe(self):
+        """
+        Verifies raw returns for a DataFrame.
+        """
+        price = self.data.iloc[0:5, 0:5]
+        test1 = raw_return(price)
+        test1_actual = pd.DataFrame([(np.nan, np.nan, np.nan, np.nan, np.nan), (0.008997, -0.002826, 0.0033758,
+                                                                                0.003779, 0.001662),
+                                     (-0.030037, -0.019552, -0.0002804, -0.025602, -0.022719), (0.007327, 0.000867,
+                                                                                                -0.000187, -0.006182,
+                                                                                                0.001045),
+                                     (-0.007754, -0.006930, 0.000748, -0.002333, -0.005610)], columns=self.col5,
+                                    index=self.idx5)
```

Comment:
A better layout can be used:
<pre>
pd.DataFrame([(np.nan, np.nan, np.nan, np.nan, np.nan), 
              (0.008997, -0.002826, 0.0033758, 0.003779, 0.001662),
              (-0.030037, -0.019552, -0.0002804, -0.025602, -0.022719),
              (0.007327, 0.000867, -0.000187, -0.006182, 0.001045),
              (-0.007754, -0.006930, 0.000748, -0.002333, -0.005610)] ...
</pre>